tests/integration-tests/test-suites/eventProcessedByFirehoseAudit.ts

```diff
@@ -0,0 +1,78 @@
+import { invokeLambdaFunction } from '../support/utils/aws/lambda/invokeLambda'
+import { getEnv } from '../support/utils/getEnv'
+import mockSQSEvent from '../support/test-auth-event.json'
+import { randomUUID } from 'crypto'
+import { cloudwatchLogFilters } from '../constants/cloudWatchLogFilters'
+import { checkSplunkBucketForEventId } from '../support/utils/aws/s3/checkSplunkBucketForEventId'
+import { baseEvent } from '../constants/baseEvent'
+import { getAuditEvent } from '../support/utils/aws/s3/returnS3FileContents'
+
+import {
+  NestedKeyOf,
+  obfuscateSpecifiedProps
+} from '../support/utils/obfuscateSpecifiedProps'
+import { AuditEvent } from '../support/types/auditEvent'
+import { SQSEvent } from 'aws-lambda'
+import { logSuccessForEventIdInLogGroup } from '../support/utils/helpers'
+import { filterCloudWatchLogs } from '../support/utils/aws/cloudwatch/cloudWatchGetLogs'
+
+describe('events processed by firehose', () => {
+  let eventId: string
+  let messageId: string
+
+  beforeAll(async () => {
+    eventId = randomUUID()
+    messageId = randomUUID()
+
+    const event: SQSEvent = {
+      Records: [
+        {
+          ...mockSQSEvent.Records[0],
+          messageId: messageId,
+          body: JSON.stringify({
+            ...baseEvent,
+            event_id: eventId
+          })
+        }
+      ]
+    }
+
+    await invokeLambdaFunction(
+      getEnv('AUDIT_MESSAGE_DELIMITER_FUNCTION_NAME'),
+      event
+    )
+    console.log(`Event ID for ${baseEvent.event_name}: ${eventId}`)
+  })
+  // Hits lambda directly - Remove and hit infra firehose function instead
+
+  /*
+    Test logs of everything downstream. First should be correct for 1st lambda
+    Replace below Splunk references with correct Audit components:
```

Comment:
These comments just look like notes for something you were doing earlier, can you remove them? We try to avoid comments as much as possible, because they're not often useful and they get out of date quickly.